scripts/ivy/fix-tsconfig.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
```

Comment:
Can you add a couple comments inline here to explain what is being replaced and removed?  I know the comments are inline in the shell script, but if someone looks here, it will help to explain in place what the script does.